src/migration/1721903257825-Vordr.ts

```diff
@@ -0,0 +1,43 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Vordr1721903257825 implements MigrationInterface {
+  name = 'Vordr1721903257825'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "user" ADD "flags" jsonb NOT NULL DEFAULT '{"trustScore": 1, "vordr": false}'`);
+    await queryRunner.query(`ALTER TABLE "comment" ADD "flags" jsonb NOT NULL DEFAULT '{"vordr": false}'`);
+
+    await queryRunner.query(`CREATE INDEX "IDX_user_flags_vordr" ON post USING HASH (((flags->'vordr')::boolean))`);
+    await queryRunner.query(`CREATE INDEX "IDX_comment_flags_vordr" ON post USING HASH (((flags->'vordr')::boolean))`);
+
+    await queryRunner.query(`CREATE OR REPLACE FUNCTION comment_flags_update_vordr()
+RETURNS TRIGGER AS $$
+BEGIN
+    -- Check if the 'vordr' key in the flags JSONB column has changed
+    IF (NEW.flags ->> 'vordr')::boolean IS DISTINCT FROM (OLD.flags ->> 'vordr')::boolean THEN
+        UPDATE comment
+        SET flags = jsonb_set(flags, '{vordr}', to_jsonb((NEW.flags ->> 'vordr')::boolean))
+        WHERE "userId" = NEW.id;
+    END IF;
+    RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;`);
+
+    await queryRunner.query(`CREATE TRIGGER user_flags_vordr
```

Comment:
you can use `CREATE OR REPLACE TRIGGER` as well

Reply:
Ah, yeah. Added that.